### ISSUE TITLE
Reproduce the results of UperNet-50/101

### ISSUE THREAD

**Issue by fanyix** [CSAILVision/semantic-segmentation-pytorch#98]
Hi, really nice repo, thanks for sharing!

One thing I noticed is that it seems the provided pretrained model (http://sceneparsing.csail.mit.edu/model/) for UperNet-50 and UperNet-101 cannot reproduce the results shown in the table of the repo page. 

![image](https://user-images.githubusercontent.com/6924959/43931216-510c5566-9bf2-11e8-9beb-19bc88c8ef60.png)

Specifically, for the single scale testing results for UPerNet-50, I got [0.3918/79.06] for Mean IoU/Pixel Accuracy. Whereas for UPerNet-101, I got [0.4061/79.95]. Is there an explanation for this? 

I simply downloaded the dataset and pretrained model, and then evaluated using the provided command -- **_python3 eval.py --id MODEL_ID --suffix SUFFIX 
--arch_encoder resnet50 --arch_decoder upernet --padding_constant 32_**

**Comment by Jongchan** [comment]
How many GPUs did you use? I have a similar problem.
I used 4 GTX1080Ti, and have 2 images per GPU. Then I got 0.3757 | 78.59% in UperNet-50.

**Comment by fanyix** [comment]
I am not even training, I am using the provided evaluation script to evaluate the provided pre-trained models from: http://sceneparsing.csail.mit.edu/model/

**Comment by SystemErrorWang** [comment]
I also tried evaluating the upernet 101 model with the pre_trained model provided here, the result is 0.392/78.88. I am working on centos, cuda9.0, cudnn 7.1.4, pytorch 0.4.1
Maybe there's minor errors/differences when working on different devices?

**Comment by hangzhaomit** [comment]
We updated the model weights and this problem is resolved.

**Comment by ThorJonsson** [comment]
Hi @hangzhaomit do you have a working link to these model weights that you can provide?
